src/lib/float32_truncate.js

```diff
@@ -8,14 +8,25 @@
 
 'use strict';
 
+function truncateFloat32(arrayIn, len) {
+    var arrayOut = new Float32Array(len);
+    for(var i = 0; i < len; i++) arrayOut[i] = arrayIn[i];
+    return arrayOut;
+}
+
+function truncateFloat64(arrayIn, len) {
+    var arrayOut = new Float64Array(len);
+    for(var i = 0; i < len; i++) arrayOut[i] = arrayIn[i];
+    return arrayOut;
+}
+
 /**
- * Truncate a Float32Array to some length. A wrapper to support environments
- * (e.g. node-webkit) that do not implement Float32Array.prototype.slice
+ * Truncate a typed array to some length.
+ * For some reason, ES2015 Float32Array.prototype.slice takes
+ * 2x as long, therefore we aren't checking for its existence
  */
-module.exports = function truncate(float32ArrayIn, len) {
-    // for some reason, ES2015 Float32Array.prototype.slice takes 2x as long...
-    // therefore we aren't checking for its existence
-    var float32ArrayOut = new Float32Array(len);
-    for(var i = 0; i < len; i++) float32ArrayOut[i] = float32ArrayIn[i];
-    return float32ArrayOut;
+module.exports = function truncate(arrayIn, len) {
```

Comment:
Nicely done here.

Let's change the file name though to `typed_array_truncate.js` (or something better?))